### PR TITLE
Initial work on azure-pipelines based builds

### PR DESCRIPTION
**Pull request by mariusvniekerk**
This should allow us to run staged recipes pull requests against azure

### CODE REVIEW

**Comment by mariusvniekerk** [comment]
Disabled build status reporting for azure for now.  Once it stabilizes we can turn it back on.

File: .azure-pipelines/azure-pipelines-osx.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+      source run_conda_forge_build_setup
+      conda update --yes --quiet --override-channels -c conda-forge -c defaults --all
+    env: {
+      OSX_FORCE_SDK_DOWNLOAD: "1"
+    }
+    displayName: Configure conda and conda-build
+
+  - script: |
+      # Run the travis script to build
+      source ./.travis_scripts/create_feedstocks;
```
Comment (isuruf) [inline .azure-pipelines/azure-pipelines-osx.yml]: This script creates feedstocks

Comment (mariusvniekerk) [reply .azure-pipelines/azure-pipelines-osx.yml]: Fixed

**Comment by scopatz** [comment]
Thanks @mariusvniekerk!

**Comment by jakirkham** [comment]
This is great! Thanks for working on this, @mariusvniekerk. 😄

File: .azure-pipelines/azure-pipelines-osx.yml (hunk 1 above)
Comment (jakirkham) [reply .azure-pipelines/azure-pipelines-osx.yml]: Do we want to explore migrating feedstock creation to Azure?

**Pull request merged**